### DISCUSSION TITLE
Alien wires are not resizeable horizontally

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14620] [Unstable feedback]
Alien wires are not resizeable horizontally
Make them resizeable horizontally

![image](https://github.com/user-attachments/assets/3b78131c-275e-4926-b703-70a2d2200491)

**Comment by Regalis11** [comment]
They are not supposed to be resizeable, you use them to wire devices the same way as the "normal" wires.

**Reply by ubertpendragon** [reply]
They exist as both wires and decorative items

![image](https://github.com/user-attachments/assets/7339d918-5f42-463b-957d-73fd3857997a)

![image](https://github.com/user-attachments/assets/35dfda27-03e0-4369-a622-828bff15c1e1)

**Reply by Regalis11** [reply]
Those are not decorative items, they're the exact same items you see in the wiring panel. Although I suppose the sprite not looking like a loop of wire like the other wires makes it confusing.

**Reply by ubertpendragon** [reply]
Guess they shouldnt appear in the editor then

Although I wouldnt mind a structure with resizehorizontal="true"

**Comment by Regalis11** [comment]
This has been addressed in our private repo now (by giving the items a different sprite to make it clearer these are not decorative structures but "actual wires"), fix coming up in the next unstable build.
